configs/airflow.production.cfg

```diff
@@ -0,0 +1,993 @@
+[core]
+# The folder where your airflow pipelines live, most likely a
+# subfolder in a code repository. This path must be absolute.
+dags_folder = /home/datamade/la-metro-dashboard/dags
+
+# The folder where airflow should store its log files
+# This path must be absolute
+base_log_folder = /var/log/la-metro-dashboard/airflow
+
+# Airflow can store logs remotely in AWS S3, Google Cloud Storage or Elastic Search.
+# Set this to True if you want to enable remote logging.
+remote_logging = True
+
+# Users must supply an Airflow connection id that provides access to the storage
+# location.
+remote_log_conn_id = S3Conn
+remote_base_log_folder = s3://la-metro-dashboard/production
```

Comment:
This is the only thing I changed. Are there other variables I should change here?

Reply:
I don't think so! We had to ratch down `parallelism` and `dag_concurrency` from the defaults because the staging server didn't have much RAM, but I actually think it's probably better to keep those at low values, particularly since it's not very important for these tasks to run in parallel.